testcases/multizone_office_simple_hydronic/doc/index.html

```diff
@@ -0,0 +1,1206 @@
+</li>
+<li>
+<code>bms_oveTSupAhuCooSz_u</code> [K] [min=273.15, max=293.15]: South zone AHU cooling water supply temperature setpoint
+</li>
+<li>
+<code>bms_oveTSupAhuHeaNz_u</code> [K] [min=293.15, max=323.15]: North zone AHU heating water supply temperature setpoint
+</li>
+<li>
+<code>bms_oveTSupAhuHeaSz_u</code> [K] [min=293.15, max=323.15]: South zone AHU heating water supply temperature setpoint
+</li>
+<li>
+<code>bms_oveTSupAhuNz_u</code> [K] [min=289.15, max=298.15]: North zone AHU air supply temperature setpoint
+</li>
+<li>
+<code>bms_oveTSupAhuSz_u</code> [K] [min=289.15, max=298.15]: South zone AHU air supply temperature setpoint
+</li>
+<li>
+<code>bms_oveTSupEmiCooNz_u</code> [K] [min=273.15, max=293.15]: North zone cooling emission circuit supply temperature setpoint
+</li>
+<li>
+<code>bms_oveTSupEmiCooSz_u</code> [K] [min=273.15, max=293.15]: Southh zone cooling emission circuit supply temperature setpoint
+</li>
+<li>
+<code>bms_oveTSupEmiHeaNz_u</code> [K] [min=293.15, max=323.15]: North zone heating emission circuit supply temperature setpoint
+</li>
+<li>
+<code>bms_oveTSupEmiHeaSz_u</code> [K] [min=293.15, max=323.15]: South zone heating emission circuit supply temperature setpoint
+</li>
+<li>
+<code>bms_oveTZonSetMaxNz_u</code> [K] [min=288.15, max=303.15]: North zone maximum (cooling) zone temperature setpoint
+</li>
+<li>
+<code>bms_oveTZonSetMaxSz_u</code> [K] [min=288.15, max=303.15]: South zone maximum (cooling) zone temperature setpoint
+</li>
+<li>
+<code>bms_oveTZonSetMinNz_u</code> [K] [min=288.15, max=303.15]: North zone minimum (heating) zone temperature setpoint
+</li>
+<li>
+<code>bms_oveTZonSetMinSz_u</code> [K] [min=283.15, max=303.15]: South zone minimum (heating) zone temperature setpoint
+</li>
+<li>
+<code>bms_oveValPosAhuCooNz_u</code> [1] [min=0.0, max=1.0]: North zone AHU cooling circuit mixing valve position setpoint
+</li>
+<li>
+<code>bms_oveValPosAhuCooSz_u</code> [1] [min=0.0, max=1.0]: South zone AHU cooling circuit mixing valve position setpoint
+</li>
+<li>
+<code>bms_oveValPosAhuHeaNz_u</code> [1] [min=0.0, max=1.0]: North zone AHU heating circuit mixing valve position setpoint
+</li>
+<li>
+<code>bms_oveValPosAhuHeaSz_u</code> [1] [min=0.0, max=1.0]: South zone AHU heating circuit mixing valve position setpoint
+</li>
+<li>
+<code>bms_oveValPosEmiCooNz_u</code> [1] [min=0.0, max=1.0]: North zone cooling emission circuit mixing valve position setpoint
+</li>
+<li>
+<code>bms_oveValPosEmiCooSz_u</code> [1] [min=0.0, max=1.0]: South zone cooling emission circuit mixing valve position setpoint
+</li>
+<li>
+<code>bms_oveValPosEmiHeaNz_u</code> [1] [min=0.0, max=1.0]: North zone heating emission circuit mixing valve position setpoint
+</li>
+<li>
+<code>bms_oveValPosEmiHeaSz_u</code> [1] [min=0.0, max=1.0]: South zone heating emission circuit supply mixing valve position setpoint
+</li>
+</ul>
+<h4>Outputs</h4>
```

Comment:
Just want to make sure it is on purpose to not have any flow meters.  Is this typical for such a building? It might be.  I noticed the nominal airflow for the AHUs is given in the above sections in the doc.  Should they similarly be noted for the pumps?

Reply:
Normally we do not encounter those. We may encounter some calorimeters per circuit that can give you a rough estimation of the flow, but is not normally the case. Nominal flows can be encounter in some as-built plans.

Reply:
Ok will leave as is since not usually encountered.